helm-chart/kuberay-operator/values.yaml

```diff
@@ -53,3 +53,13 @@ batchScheduler:
 # Set up `securityContext` to improve Pod security.
 # See https://github.com/ray-project/kuberay/blob/master/docs/guidance/pod-security.md for further guidance.
 securityContext: {}
+
+# When singleNamespaceInstall is true:
+# - the chart can be installed by users with permissions to a single namespace only
+#   (this excludes the CRDs which can only be installed at cluster-scope)
+# - the KubeRay operator will only listen to the resource
+#   events from the operator's namespace by default
+singleNamespaceInstall: false
+
+# kuberay operator will only watch the resource events from the "watchNamespace" namespace.
```

Comment:
Add some comments to describe the relationship between `singleNamespaceInstall` and `watchNamespace`.

Reply:
added